spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -90,12 +90,35 @@
     end
   end
 
-  describe "#unschedule" do
+  describe "#schedule" do
+    let(:step_by_step_page) { create(:draft_step_by_step_page) }
+
     before :each do
-      stub_user.permissions << "Scheduling"
+      stub_scheduling_permissions
+      stub_publishing_api_for_scheduling
+    end
 
-      allow(Services.publishing_api).to receive(:get_content)
-        .and_return(content_item(step_by_step_page))
+    it "sets `scheduled_at` to a datetime" do
+      expect(step_by_step_page.scheduled_at.class.name).to eq 'NilClass'
```

Comment:
This line isn't needed, because it's testing the set-up of the factory, and not the action of the controller.